libs/pipingapp/src/lib/utils-table/generateCommaSeperatedStringArrayColumn.ts

```diff
@@ -0,0 +1,15 @@
+export function generateCommaSeperatedStringArrayColumn(
+  stringArray: string[],
+  maxLength?: number
+): string {
+  if (stringArray.length === 0) {
+    return '';
+  }
+  const length = maxLength ?? 3;
+  let commaString = stringArray.slice(0, length).join(', ');
```

Comment:
Are you sure these files are being used?

Reply:
not yet